Documentation/ceph-upgrade.md

```diff
@@ -52,7 +52,7 @@ With this upgrade guide, there are a few notes to consider:
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
 another are as simple as updating the common resources and the image of the Rook operator. For
-example, when Rook v1.6.1 is released, the process of updating from v1.6.0 is as simple as running
+example, when Rook v1.6.1 is released, the process of updating from v1.6.1 is as simple as running
```

Comment:
I think maybe the easiest way for us to handle this is to make this change?

```suggestion
example, when Rook v1.6.1 is released, the process of updating from v1.6.0 is as simple as running
```

In the future, when we replace v1.6.1 with v1.6.2, the example will still show the process of updating from v1.6.0.